lib/generateMarkdown.js

```diff
@@ -73,7 +73,7 @@ function generateMarkdown(options) {
         }
 
         // Render section for each property
-        md += createPropertiesDetails(schema, title, options.headerLevel + 1, options.suppressWarnings);
+        md += createPropertiesDetails(schema, '', options.headerLevel + 1, options.suppressWarnings);
```

Comment:
Additionally, this looks weird for this scenario....we end up seeing property names with a leading period like this:

### .bufferView

The index of the bufferView.

* **Type**: `integer`